client/src/components/PlaceTile.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const PlaceTile = (props) => {
+  const { distance, id, image, location, name, price, rating, url } = props
+  const getLocation = location.join(' ')
+  
+  return (
+    <div className="cell">
+      <div className="card">
+        <div className="card-divider">
+          <h2>{name}</h2>
+        </div>
+        <div className="card-section">
+          <div className="card-image">
+            <img src={image} alt="Business Header Image"/>
+          </div>
+        </div>
+        <div className="card-section">
+        <p>Price: {price}</p>
+        <p>Rating: {rating}</p>
+        <p>Address: {getLocation}</p>
+        <a href={url}>See their Business on Yelp</a>
+        <p>Distance from you: {distance}</p>
+        </div>
```

Comment:
watch your div indentation on this last div, looks like the child elements need to be tabbed in